src/Monolog/Handler/SyslogUdpHandler.php

```diff
@@ -93,10 +94,13 @@ protected function makeCommonSyslogHeader(int $severity): string
             $hostname = '-';
         }
 
-        $date = $this->getDateTime();
-
         if ($this->rfc === self::RFC3164) {
-            return "<$priority>" .
+	    	$datetime->setTimezone(new \DateTimeZone('UTC'));
```

Comment:
The indent is off here if you can use spaces that'd be great.

Reply:
Hello! Thanks for the hint. I think it's fixed.